Src/Controller/ApiController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Aot\Controller;
+
+use Aot\Model\StrategyModel;
+
+class ApiController
+{
+    /**
+     * lead to the service
+     * @param $parameters
+     * @return array
+     * @throws \Exception
+     */
+    public function sendToService($parameters)
+    {
+        if (array_key_exists('format', $parameters) && array_key_exists('method', $parameters)) {
```

Comment:
`method` should be covered by the router, `format` is for the logic and it is ok to have it here